src/Tests/Microsoft.NET.Publish.Tests/Microsoft.NET.Publish.Tests.csproj

```diff
@@ -20,6 +20,7 @@
 
   <ItemGroup>
     <PackageReference Include="FluentAssertions.Json" Version="$(FluentAssertionsJsonVersion)" />
+    <PackageReference Include="System.Reflection.MetadataLoadContext" Version="5.0.0" />
```

Comment:
The version should go into the https://github.com/dotnet/sdk/blob/2829afcb4f31f65f195f5b4dc82b4d84ae096610/eng/Versions.props#L141 into the test dependencies section.
@sbomer should there be some DARC magic configured to get the latest? I don't think we want to hardcode 5.0.0 forever....

Reply:
Yes, it should have an entry in https://github.com/dotnet/sdk/blob/master/eng/Version.Details.xml, then it should get updated when runtime bits flow.

Reply:
How does DARC figure out which component it is - is there some kind of pattern in the name of the version property? And how would it know that it comes from the runtime repo?
Just curious - if it works... I don't complain :wink:

Reply:
There's a naming pattern - the property should be <SystemReflectionMetadataLoadContextVersion>` in Versions.props.

If you run "darc add-dependency -n System.Reflection.MetadataLoadContext -r https://github.com/dotnet/runtime -t toolset -v <version> -c <commit>" it will locally add the property for you and update Version.Details.xml:
```
<Dependency Name="System.Reflection.MetadataLoadContext" Version="...">
      <Uri>https://github.com/dotnet/runtime</Uri>
      <Sha>...</Sha>
</Dependency>
```